PROPOSALS.md

```diff
@@ -3,6 +3,14 @@
 Want to make a change to `base`? Great! We'd love to hear your suggestion.
 Just follow the steps outlined here so that your idea doesn't lose traction!
 
+TL;DR Quick overview of the proposal lifecycle:
+
+1. [Create an issue][open-issue] with your idea.
+2. Make [impact assessment](#impact-assessments) for breaking changes.
+3. Discuss and refine proposal.
+4. Wait for the vote.
+5. If accepted, implement the proposal.
```

Comment:
I think I'd rather have `Discuss and refine proposal` as an implied step between each of these - I would personally rather evaluate a small idea/proposal for feasibility before jumping into the next step, esp an impact assessment.